conf/messages

```diff
@@ -21,6 +21,7 @@ site.opensInNewTab = (opens in new tab)
 site.item = item
 site.noResultsFound = No results found
 site.divider = or
+site.confirmCodes = Confirm codes
```

Comment:
I don't think this relates to `site` per se, it seems very specific to the commodity code page.

If you move this down to the rest of the `confirmCommodityCode` messages and rename to:
```suggestion
confirmCommodityCode.button = Confirm codes
```